### PR TITLE
Add schema id and byte id in parameter to fix exi header encoding.

### PR DESCRIPTION
**Pull request by Nawroski**

### CODE REVIEW

File: src/main/java/com/siemens/ct/exi/main/cmd/EXIficientCMD.java [hunk 1]
```diff
@@ -286,7 +288,14 @@ else if (NO_SCHEMA.equalsIgnoreCase(argument)) {
...
 			}
 			// ### OPTIONS
-			else if (OPTION_STRICT.equalsIgnoreCase(argument)) {
+			else if (SCHEMA_ID.equalsIgnoreCase(argument)) {
+        indexArgument++;
+				exiFactory.setSchemaId(args[indexArgument]);
+      } else if (BYTE_ID.equalsIgnoreCase(argument)) {
+        indexArgument++;
+        int byteVal = Integer.valueOf(args[indexArgument]);
+        exiFactory.setByteId(byteVal);
```
Comment (danielpeintner) [inline src/main/java/com/siemens/ct/exi/main/cmd/EXIficientCMD.java:297]: This does not exist?

File: src/main/java/com/siemens/ct/exi/main/cmd/EXIficientCMD.java [hunk 2]
```diff
@@ -102,6 +102,8 @@ public class EXIficientCMD {
 	public static final String XSD_SCHEMA = "-" + SchemaOption.xsdSchema;
 	public static final String SCHEMA = "-" + SchemaOption.schema;
 
+	public static final String SCHEMA_ID = "-schemaId";
+	public static final String BYTE_ID = "-byteId";
```
Comment (danielpeintner) [inline src/main/java/com/siemens/ct/exi/main/cmd/EXIficientCMD.java:106]: This option does not exit in the [EXI](https://www.w3.org/TR/exi/) specification?

**Review by danielpeintner — CHANGES_REQUESTED** [review]

**Comment by danielpeintner** [comment]
In general I think it is better to first discuss/describe what you try to achieve in an issue...

**Comment by Nawroski** [comment]
> In general I think it is better to first discuss/describe what you try to achieve in an issue...

I want to add an option to force the value for the header, this is project specific actually so it might not match the EXI specifications.

**Comment by danielpeintner** [comment]
This change simply does not compile for others.. there is no such call `exiFactory.setByteId(byteVal)`
If you need extensions I think you are better suited to extend the class in your repo..

**Comment by Nawroski** [comment]
I opened a pull request on exificient-core to add those symbols, but i understand if you do not want those extensions in your master branch.

**Comment by danielpeintner** [comment]
There is no such *open* PR. You opened https://github.com/EXIficient/exificient-core/pull/9 and closed it again.
Anyhow.. We would not accept settings that do not belong to the EXI spec.

**Pull request closed without merging**